src/main/java/duke/tools/Ui.java

```diff
@@ -55,24 +55,28 @@ public String testReadCommand() {
         String prompt_message = "Please input your command: ";
         return prompt_message;
     }
+
+    public static void printMessage(String message) {
+        System.out.println(DIVIDER + "\t" + message + DIVIDER);
+    }
 }
 
 //Code to expand on in the future
```

Comment:
Do you think there might be a better way to manage your work in progress code besides committing them to your git history? 🙂

Reply:
any suggestions? another branch perhaps

Reply:
try this link: jibble.org/stfw